src/components/notes/editor-view.jsx

```diff
@@ -37,6 +36,14 @@ function EditorView(props) {
     [onTitleChange]
   )
 
+  const onContentChange = useMemo(
+    () => {
+      const nullCallback = () => {}
+      return props.onContentChange || nullCallback
+    },
+    [props.onContentChange]
+  )
+
```

Comment:
Why is this better than straight up using the prop?

Reply:
None, you're right. Let me change that.
Thank you